members/_posts/2016-10-04-lewic.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : lewic.png

```

Comment:
Ok, I am fine with the auto-scaling (and you can reduce the size of yours indeed). So I'd suggest to only limit the size in KB to e.g. 50 KB, so that we keep the member list loading time under control.